pkg/controller/eventing/subscription/reconcile_test.go

```diff
@@ -380,6 +381,79 @@ var testCases = []controllertesting.TestCase{
 					},
 				}},
 		},
+	}, {
+		Name: "new subscription to K8s Service: adds status, all targets resolved, subscribers modified",
+		InitialState: []runtime.Object{
+			getNewSubscriptionToK8sService(),
+			getK8sService(),
+		},
+		ReconcileKey: fmt.Sprintf("%s/%s", testNS, subscriptionName),
+		// TODO: JSON patch is not working for some reason. Is this the array vs. non-array, or
```

Comment:
I figured out what the problem is a few days ago, if you could update these comments with a pointer to this bug:
https://github.com/kubernetes/client-go/issues/478

Then I'd rather leave them here as is for the reasons you already mentioned, it does do quite a few tests and covers most of the code.

Reply:
Done.